Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -120,6 +120,38 @@ void Compositor::did_construct_window_manager(Badge<WindowManager>)
     compose();
 }
 
+const Gfx::Bitmap& Compositor::bitmap_of_active_window(Badge<ClientConnection>)
+{
+    auto& wm = WindowManager::the();
+
+    const auto active_window = wm.active_window();
+
+    auto& active_window_frame = active_window->frame();
+    const auto active_window_frame_render_rect = active_window_frame.render_rect();
+    const auto size = Gfx::IntSize { active_window_frame_render_rect.width(), active_window_frame_render_rect.height() };
+
+    auto& screen = Screen::the();
+    const auto scale_factor = screen.scale_factor();
+
+    m_active_window_bitmap = Gfx::Bitmap::create(Gfx::BitmapFormat::BGRA8888, size, scale_factor);
```

Comment:
Creating a bitmap can fail if we run out of memory. We need to null-check the result of `Gfx::Bitmap::create()`